OktaJWT.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'OktaJWT'
-  s.version          = '1.0.0'
+  s.version          = '1.0.1'
```

Comment:
nit: Lets hold off on bumping up the version for now. This will allow us to integrate other changes and add a `CHANGELOG` before shipping a new version.

Reply:
The reason is that this library is a dependency of `okta-sdk-appauth-ios`, which also should be updated to latest Swift as soon as possible. We can't do that without updating this pod.